ui/app/templates/rules.html

```diff
@@ -229,6 +229,11 @@ <h5 title="Fallback Mapping" ng-show="rule.fallbackMapping || (show_sc && rule.s
                  help='Whether or not to check a client&#39;s 32 -> 64-bit migration opt-in status.'></rulefield>
       <rulefield showdiff="!rule_id" showsc="show_sc" rule="rule" fieldname="update_type" fieldtitle="Update type"></rulefield>
       <rulefield showdiff="!rule_id" showsc="show_sc" rule="rule" fieldname="headerArchitecture" fieldtitle="Header Architecture"></rulefield>
+      <span ng-show="rule.fallbackMapping || (!rule_id && show_sc && rule.scheduled_change !== null && rule.fallbackMapping !== rule.scheduled_change.fallbackMapping && rule.scheduled_change.fallbackMapping !== null) && rule.backgroundRate === 100" > 
```

Comment:
This condition doesn't seem to be fully working. For example, when backgroundRate is set to 0 (or even 99), the text still shows up: https://screenshots.firefoxusercontent.com/images/12650353-b623-4e76-ad64-a20f4245512d.png

You also need to account for a possible scheduled change here. The logic for showing this should be something like:
* If the Rule's backgroundRate is not 100 OR the scheduled change's backgroundRate is not 100, show the warning

Please ensure to test the following scenarios:
* Rule at 100 with no scheduled change
* Rule < 100 with no scheduled change
* Rule and scheduled change both at 100
* Rule and scheduled change both < 100
* Rule < 100, scheduled change at 100
* Rule at 100, scheduled change < 100
* Just a scheduled change (use the "Schedule a new Rule" button) at 100
* Just a scheduled change (use the "Schedule a new Rule" button) < 100

Reply:
Ahh ! thats probably because a closed parenthesis at the start of  `rule_id &&` is missing.

Does it boils down to fact that if fallbackMapping is present(either from rule or scheduled change) if  backgroundRate( either from rule or scheduled change ) is 100 then throw an error.

Reply:
Yep, that's another way to phrase it!